frontend/.eslintrc.yml

```diff
@@ -4,7 +4,6 @@
   'extends':
     [
       'react-app',
-      'eslint:recommended',
```

Comment:
I believe you've added this and I don't think it is needed.  I suggest removing it